bluepyopt/ephys/efeatures.py

```diff
@@ -22,6 +22,7 @@
 
 import logging
 
+from bluepyopt.ephys.serializer import DictMixin
```

Comment:
We'll probably need some more helper functions for serializing groups of objects, but it's basically just a list comprension: `[o.to_dict() for o in objects]`